requirements.txt

```diff
@@ -1,5 +1,6 @@
-cosl
-jenkinsapi>=0.3,<1
-ops>=2,<3
-pydantic>=1,<2
-requests>=2,<3
+cosl==0.0.7
```

Comment:
I'm wondering what the difference is in having pinned `requirements.txt` and `poetry.lock` might be.

Reply:
We don't have peotly yet in this repo, but it will do the trick for pinning deps